packages/backend/src/api/routes/health.ts

```diff
@@ -0,0 +1,24 @@
+import type { ApiRoutes } from '@vse-bude/shared';
+import { Router } from 'express';
+import { PrismaClient } from '@prisma/client';
+import { RedisStorageService } from 'services/redis-storage';
+
+export const initHealthRoutes = (path: ApiRoutes) => {
+  const router = Router();
+  let status = 200;
+
+  const prisma = new PrismaClient({
+    log: ['query', 'info', 'warn', 'error'],
+  });
+
+  try {
+    prisma.user.findMany();
```

Comment:
If error thrown it will not be handled because there is no `await`. Also retrieving all users can be very slow, we need status as soon as possible. probably use something from this https://stackoverflow.com/questions/69568781/how-to-create-a-custom-health-check-for-prisma-with-nestjs-terminus